MarsOnboardingTask/Features/Languages.feature

```diff
@@ -0,0 +1,26 @@
+Feature: This test suite runs language related test scenarios
+
+ 
+
+Scenario: 1. Verify user is able to add languages with language level
+Given User logs into Mars application
+When User adds 'French' language with 'Basic' language level into the languages list
+Then The 'French' language with 'Basic' language level should be added to languages list
+
+Scenario: 2. Verify User is able to update an existing language with a new language level
```

Comment:
please increase your coverage you need to think about the following scenarios 
• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

- E.g of above scenarios
- Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.
- 
- The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).
- 
- Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).
- 
- If some scenarios does not make sense to add you can you should know why it does not make  sense to add.

Reply:
updated as per the review comments